### DISCUSSION TITLE
Duda observer

### DISCUSSION THREAD

**Discussion by Franpanozzo** [dds-jv/foro#276] [Consultas de objetos] [answered]
Buenas, cuando aplicamos este patron tratamos de no modificar al objeto observado?? En un escenario en el que te afecte ese orden para los demas observers no convendria, como dice el [documento](https://docs.google.com/document/d/1UwTcRLugqDgZuqfWvOxckwk27UBjDo70AF1znzX24QM/edit#heading=h.y04j3mise0wn). Pero si este no es el caso habria problema?? Saludos

**Comment by gastonprieto** [comment, marked as answer]
buenas @Franpanozzo 

Es mas que nada un warning o algo para tener en cuenta al modelar con observers, de que en general se recomiendan que no afecten al observado. Pero se puede hacer si ves que no hay problemas, solo hay que tenerlo en cuenta c/vez que agregues un nuevo observador que podría verse afectado.

**Reply by Franpanozzo** [reply]
Buenisimo gracias gaston!!
